gui-wm/hikari/metadata.xml

```diff
@@ -20,5 +20,6 @@
         <flag name="layershell">add support for wayland apps using layer shell</flag>
         <flag name="screencopy">add support for screen grabbing</flag>
         <flag name="man">build and install man pages</flag>
+        <flag name="virtual">enable virtual input/output support, needed for VNC</flag>
```

Comment:
What does this **exactly** do? Is it like a virtual keyboard, or a requirement for any remote desktop session? Just saying this USE flag sounds weird and there's probably a better alternative already being used in ::gentoo repo.

Reply:
renamed to virtual-io and added a few more words
its needed for remote IO using WayVNC or other wayland VNC softwares